scripts/ci

```diff
@@ -5,3 +5,4 @@ cd $(dirname $0)
 
 ./build
 ./validate
+./test
```

Comment:
Should we change the order?
```suggestion
./validate
./test
./build
```

Reply:
I'm okay with either. Anyone else wanna chime in on this or expand on which order would be best?

Reply:
Updated to your suggestion.